src/renderer/pages/project/components/product_tabs.vue

```
@@ -140,7 +140,7 @@
 };
 
 const downloadFile = async (fileType: string, mimeType: string, fileName: string) => {
-  const buffer = await window.electronAPI.mulmoHandler("downloadFile", projectId.value, fileType);
+  const buffer = (await window.electronAPI.mulmoHandler("downloadFile", projectId.value, fileType)) as BlobPart;
```

ESLint

[error] 'BlobPart' is not defined.

(no-undef)